src/applications/simple-forms/21P-601/config/form.js

```diff
@@ -48,7 +49,7 @@ const formConfig = {
   prefillTransformer,
   savedFormMessages: {
     notFound:
-      'Please start over to apply for Application for Accrued Amounts Due a Deceased Beneficiary.',
+      'Please start over to apply for Application for Apply for accrued benefits online.',
```

Comment:
this sounds a little strange should probably be 

"Please start over to apply for accrued benefits online."